weaviate/gql/get.py

```diff
@@ -928,17 +929,43 @@ def with_hybrid(
         self._contains_filter = True
         return self
 
-    def build(self) -> str:
+    def with_alias(
+        self,
+        alias: str,
+    ):
+        """Gives an alias for the 'class_name'. Needs to be used if 'multi_get' requests the same 'class_name' twice.
```

Comment:
I would say that it is an alias for the query. You can use the same class name with different aliases, so maybe to avoid any confusion it would be better to say `alias for the query`.

Ex: `language_en` could be an alias, and it is unrelated to the class at all.